src/modules/_nomodule/PixelManipulation.js

```diff
@@ -86,10 +86,8 @@ module.exports = function PixelManipulation(image, options) {
               y
             );
 
-            pixels.set(x, y, 0, pixel[0]);
-            pixels.set(x, y, 1, pixel[1]);
-            pixels.set(x, y, 2, pixel[2]);
-            pixels.set(x, y, 3, pixel[3]);
+            require('../../util/pixelSetter.js')(x, y, pixel, pixels);
```

Comment:
At the top

Reply:
:laughing: so basically you are suggesting to move require to the top and the const to be added instead of var. Anything else ? Thanks a lot :hugs:

Reply:
Yes, That's it. TY